source/support/connect-to-govwifi-using-a-windows-device.html.erb

```diff
@@ -0,0 +1,134 @@
+      <%= image_tag "troubleshooting/windows-7-step-6.png", class: "img-fluid" %>
+
+      <p>Step 7 - Select the <strong>Security</strong> tab and select <strong>Settings</strong>.</p>
+      <%= image_tag "troubleshooting/windows-7-step-7.png", class: "img-fluid" %>
+
+      <p>Step 8 - Tick the <strong>Connect to these servers box</strong>. In the text box below it, enter <strong>wifi.service.gov.uk</strong>.</p>
+
+      <p>Step 9 - In the <strong>Trusted Root Certification Authorities</strong> list, select <strong>GeoTrust Primary Certification Authority - G3.</strong></p>
+
+      <p>Step 10 - Tick <strong>Enable Identity Privacy</strong> and enter <strong>anonymous</strong> to prevent your device from broadcasting your GovWifi username.</p>
+
+      <p>Step 11 - Select <strong>Configure.</strong></p>
+      <%= image_tag "troubleshooting/windows-7-step-11.png", class: "img-fluid" %>
+
+      <p>Step 12 - Untick <strong>Automatically use my Windows logon name</strong>.</p>
+
+      <p>Step 13 - Select <strong>OK</strong>, then <strong>OK</strong> again to go back to the <strong>GovWifi Wireless Network Properties</strong> window.</p>
+      <%= image_tag "troubleshooting/windows-7-step-13.png", class: "img-fluid" %>
+
+      <p>Step 14 - Select <strong>Advanced settings</strong>.</p>
+      <%= image_tag "troubleshooting/windows-7-step-14.png", class: "img-fluid" %>
+      <p>Step 15 - Select <strong>User authentication</strong> from the dropdown menu under <strong>Specify authentication mode</strong>, then select <strong>Save credentials</strong>.</p>
+
+      <%= image_tag "troubleshooting/windows-7-step-15.png", class: "img-fluid" %>
+
+      <p>Step 16 - Enter your GovWifi <strong>username</strong> and <strong>password</strong>.</p>
+      <p>Your password is case sensitive. It's 3 short words without spaces. The first letter of each word is capitalised.</p>
+      <p>Step 17 - Select <strong>OK</strong> to close all windows.</p>
+      <p>Step 18 - Select the wifi logo at the bottom right of your screen and select <strong>GovWifi</strong>.</p>
+      <p>You should now be connected to GovWifi.</p>
+
+      <h2 id="support-windows-7">Support for Windows 7</h2>
+      <p>If you can't see 'GeoTrust Primary Certification Authority - G3' in the Trusted Root Certification Authorities list you might not have the right certificate installed on your device. If you manage your own device, you can download it from the web.</p>
+
+      <p>Step 1 - In your web browser, navigate to: <%= link_to "https://www.geotrust.com/resources/root_certificates/certificates/Geotrust_PCA_G3_Root.pem", "https://www.geotrust.com/resources/root_certificates/certificates/Geotrust_PCA_G3_Root.pem" %></p>
+      <p>Step 2 - Go to your Start menu. Select 'Accessories' and choose 'Notepad'.  Copy everything from BEGIN CERTIFICATE to END CERTIFICATE into your notepad, and save the file on your C: drive as certificate.pem</p>
```

Comment:
Maybe certificate.pem should be in code tags?

Reply:
Cool all the code is now wrapped in code tags.